app.js

```diff
@@ -14,5 +15,10 @@ app.use(MODELS_MOUNT_PATH, express.static(MODELS_DIR));
 
 app.get('/', (req, res) => res.sendFile(INDEX_PATH));
 
-// eslint-disable-next-line no-console
-app.listen(PORT, () => console.log(`Listening on port ${PORT}`));
+reload(app)
+  .then(() => {
+    // eslint-disable-next-line no-console
```

Comment:
Hmm is there a way of logging things to stdout/stderr that the linter does look favorably upon?

Reply:
Good point, I'll see what I can do to make ESLint happy without losing the logging ability.
We could use either `nclr`, `winstone`, `debug`, `signale` or others.
I'm not sure what the best approach would be for browser logging as I haven't used logging libraries there (other than using `console....()`).

Reply:
sg! Not blocking this PR - we should just keep it in mind and adopt one of these that makes the most sense!

Reply:
Yeah, I've opted for `nclr` for the `app.js` which works wonders and don't require ESLint directives.